src/app/service/request.tsx

```diff
@@ -0,0 +1,10 @@
+export default function useRequests(){
+    const endpoints={
+        getAllProducts:`${process.env.NEXT_BACKEND_API_URL}/product`,
+        createProduct:`${process.env.NEXT_BACKEND_API_URL}/product`,
+        deleteProductById:`${process.env.NEXT_BACKEND_API_URL}/Product`,
```

Comment:
route is capitalised here alone, you can move route to a constant. 
Where ever applicable if we are using same value at multiple places move them to constants

Reply:
ok sure.